### PR TITLE
Add mapper support device data writing proposal

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind design

**What this PR does / why we need it**:
Add mapper support device data writing proposal
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by fujitatomoya — COMMENTED** [review]
really interesting feature, had a minor comment.

**Review by Shelley-BaoYue — COMMENTED** [review]
If this proposal has been completed, please remove the WIP label.

File: docs/proposals/mapper-support-devicedata-writing.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+When device property field in the device CRD is modified, the `DeviceController` on the cloud side will send the updated
+device configuration file to the `DeviceTwin` on the edge side through the cloud edge channel. Then `DeviceTwin` sends the
+device update information to Mapper, executes the `updatedev` function, and completes the update and writing of device property.
+Although this method conforms to the native definition of Kubernetes, in edge scenarios, the cloud-edge network is unstable,
+which may cause the device write control command to be unable to send to the edge. At the same time, if the device write
+request volume is large, may cause the cloud edge channel to be blocked.
+
+Therefore, this solution shown bellow refers to the way the Mapper Framework data plane processes device data and provides a
+device writing interface on the data plane. With this interface, users can directly send device write commands
+to the mapper data plane to prevent cloud edge channel blocking.
```
[the remark this answers is not in the document]
Comment (wbc6080) [reply docs/proposals/mapper-support-devicedata-writing.md:37]: Sorry for the late reply. The question you mentioned makes sense, and there should indeed be some security when sending device write requests. But I also want to point out that mapper is ultimately deployed in the cluster in the form of a pod. In order to facilitate understanding in the proposal, I directly use `nodeport` to expose services. Users can choose other safe ways in kubernetes to expose services when using it. For example, they can forward routes in the form of Ingress, allowing only safe traffic to arrive, or choose to only allow requests to be sent within the cluster.
Of course, what you put forward is also a good suggestion. I also add security improvement to the proposal as a feature to be considered for improvement in the next version. :)

**Comment by wbc6080** [comment]
> If this proposal has been completed, please remove the WIP label.

Done. Ready to be reviewed

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**